src/index.test.js

```diff
@@ -30,13 +30,53 @@ test('serviceConfig', (t) => {
             consulPrefix: 'prefix'
           }
         }
+      },
+      cli: {
+        log(message) {
+          // eslint-disable-next-line no-console
+          console.log(message);
```

Comment:
I'm console logging in this mock to emulate the behaviour of the serverless log method